### PR TITLE
Mango Improvements

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

File: src/loans/loan_state.py [hunk 1]
```diff
@@ -254,6 +253,10 @@ def process_events_to_loan_states(
             'debt': [{token: float(amount) for token, amount in loan.debt.items()} for loan in state.loan_entities.values()],
         }
     )
+
+    # TODO: Upload health ratios etc here, rn every protocol has their
```
Comment (lukaspetrasek) [inline src/loans/loan_state.py:257]: I do it during the liquidable debt computation for MF. The reason is that for both computations we need to fetch some params (collateral/debt factors, mints, decimals, liquidation params, ..). So I prefer to do it at once.

File: src/loans/mango.py [hunk 2]
```diff
@@ -157,9 +197,86 @@ def process_unprocessed_events(self):
...
+                    continue
+
+                if not quote_mint:   
+                    print(f'Unable to find quote token index map for index {order.quote_token_index}')
+                    continue
                 
+                try:
+                    # Fetch Serum order account info
+                    fetched_order = self.client.get_account_info(order.open_orders)
+                except SolanaRpcException:
```
Comment (lukaspetrasek) [inline src/loans/mango.py:226]: At this point, isn't it worth a try to fetch the orders again at least once after some short sleep?

File: src/loans/mango.py [hunk 3]
```diff
@@ -306,18 +423,32 @@ def get_mango_user_stats_df(loan_states: pd.DataFrame) -> pd.DataFrame:
...
         'debt_usd', 
         'risk_adj_debt_usd',
-        'collateral',
-        'debt',
+        'protocol', 
     ]
 
     return loan_states[wanted_cols]
 
+    # user = Column(String, primary_key=True, nullable=False)
```
Comment (lukaspetrasek) [inline src/loans/mango.py:443]: This is redundant now, isn't it?

File: src/loans/loan_state.py (hunk 1 above)
Comment (Chepelau) [reply src/loans/loan_state.py:257]: Okay, I'll remove the todo

File: src/loans/mango.py (hunk 2 above)
Comment (Chepelau) [reply src/loans/mango.py:226]: Yep

**Pull request merged**